src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -593,6 +451,159 @@ public BulkLoadRegisterSDFResponseDTO registerSdf(BulkLoadRegisterSDFRequestDTO
 		}
 	}
 
+	@Transactional(propagation = Propagation.REQUIRES_NEW)
+	public Boolean registerMol(CmpdRegMolecule mol, BulkLoadFile bulkLoadFile,Collection<BulkLoadPropertyMappingDTO> mappings,
```

Comment:
It could be argued that I refactor most of the variables in this function up  a level as they don't really need to be passed in here.  I took the short cut for now...